src/qr.rs

```diff
@@ -543,21 +545,29 @@ async fn decode_ideltachat(context: &Context, prefix: &str, qr: &str) -> Result<
         .with_context(|| format!("failed to decode {prefix} QR code"))
 }
 
-/// scheme: `DCACCOUNT:https://example.org/new_email?t=1w_7wDjgjelxeX884x96v3`
+/// scheme: `DCACCOUNT:example.org`
```

Comment:
Would make sense to document this variant in the interface repo https://github.com/deltachat/interface/blob/main/uri-schemes.md

Reply:
I opened https://github.com/deltachat/interface/pull/87